qcodes_contrib_drivers/drivers/Lakeshore/Model_625.py

```diff
@@ -0,0 +1,524 @@
+import logging
+import time
+from typing import Union, Tuple
+
+
+from qcodes import VisaInstrument
+from qcodes.utils.validators import  Numbers, Enum
+
+log = logging.getLogger(__name__)
+
+class Lakeshore625(VisaInstrument):
+    """
+    Driver for the Lakeshore Model 625 superconducting magnet power supply.
+
+    This class uses T/A and A/s as units.
+
+    Args:
+        name (str): a name for the instrument
+        address (str): VISA address of the device
+        current_ramp_limit: A current ramp limit, in units of A/s
+    """
+
+    def __init__(self, name: str, coil_constant: float,  field_ramp_rate: float, address: str=None,
```

Comment:
address cannot be None within the VisaInstrument class. I think you can just remove the `=None` part here to fix it

Reply:
Thanks for pointing out these two problems. I have fixed them and the tests pass now. Let me know if additional work needs to be done.